flink-connectors/flink-connector-kafka-0.11/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer011Test.java

```diff
@@ -61,7 +61,7 @@
  * IT cases for the {@link FlinkKafkaProducer011}.
  */
 @SuppressWarnings("serial")
-public class FlinkKafkaProducer011Tests extends KafkaTestBase {
+public class FlinkKafkaProducer011Test extends KafkaTestBase {
```

Comment:
Shouldn't this be named `*ITCase` according to the coding conventions ?

Reply:
Aren't all of the other `ITCase`s starting a Flink job? 

Regardless I named it `Test` originally because it is slightly more unit "testish" compared to `Kafka011ProducerExactlyOnceITCase`.

Reply:
I haven't looked into too many `ITCase`s but coding guidelines require unit tests to be subsecond execution speed:
>Please use unit tests to test isolated functionality, such as methods. Unit tests should execute in subseconds and should be preferred whenever possible. The name of unit test classes have to on *Test. Use integration tests to implement long-running tests.

http://flink.apache.org/contribute-code.html#coding-guidelines

Then again, I don't know how consistent the tests are.

Reply:
I'm leaning towards `*ITCase` here, since it is indeed an integration test with Kafka.